docs: add multi credential exchange definition with or condition (#168)
energywebfoundation/ssi: apps/vc-api/docs/example-exchange-definitions/multi-credential-exchange-definition-with-or-condition.md

Exchange definition requiring any or some of credential from a list of credential.

> To create `presentation-definition` which accepts any or some of the Verifiable Credentials from a list we use `group` to categories the credential to choose from. 

Reviewer: Spelling and grammar
```suggestion
To create a `presentation-definition` which accepts any or some of the Verifiable Credentials from a list, we use `group` to categorise the credentials to choose from. 
```

> The `submission_requirements` property of `presentation-definition` specifies the number of credentials need to be presented against the definition within a group. Refer [Submission Requirements](https://identity.foundation/presentation-exchange/#submission-requirements) documentation for more information.

Reviewer: ```suggestion
The `submission_requirements` property of `presentation-definition` specifies the number of credentials that need to be presented against the definition within a group.
Refer to the [Submission Requirements](https://identity.foundation/presentation-exchange/#submission-requirements) documentation for more information.
```

Use case: 

> 1. The Holder is required to present any one of the credential from the list of credential (Group A).

Reviewer: ```suggestion
1. The Holder is required to present any one of the credentials from a list of credentials (Group A).
```

* The below `presentation-definition` requires the holder to either present `PermanentResidentCard` or `ConsentCredential` for a successful verification.

> * Both the required credentials are part of `group : ["A"]` and the `submission_requirements` specifies `min` property to be `1`, which states that any one credential is required.

Reviewer: ```suggestion
* Both of the required credentials are a part of `group : ["A"]` and the `submission_requirements` specifies the `min` property to be `1`, which states that any one credential is required.
```
